optical/converter/base.py

```diff
@@ -6,11 +6,17 @@
 
 from abc import ABC, abstractmethod
 from typing import Optional
+import warnings
 
 import altair as alt
 import pandas as pd
 
-from .converter import convert_coco, convert_csv, convert_pascal, convert_sagemaker, convert_yolo
+try:
+    import tensorflow as tf
+except Exception:
+    tf = None
+
```

Comment:
1. Please don't catch blanket exceptions, it's a potential risk. In this case we should particularly catch `ImportError`.
2. Please use a boolean flag e.g.,

```
_TF_INSTALLED = True
try:
   import tensorflow as tf
except ImportError:
   _TF_INSTALLED = False
```

Reply:
Used Lazy loading for now